pkg/runtime/framework/plugins/torch/torch.go

```diff
@@ -188,3 +230,40 @@ func calculateNumProcPerNode(
 	}
 	return intstr.FromInt32(defaultCPU), false
 }
+
+// getRecipeFromArgs extracts the recipe from the distributed parameters and command line arguments.
+// TODO(Electronic-Waste): Add support for more recipes.
+func getRecipeFromArgs(numNodes int32, numProcPerNode intstr.IntOrString, _ []string) string {
+	recipe := constants.TorchTuneDefaultRecipe
```

Comment:
Can you just assign TorchTuneFullFinetuneDistributed to reduce number of constants.
```suggestion
	recipe := constants.TorchTuneFullFinetuneDistributed
```